pyproject.toml

```diff
@@ -66,7 +66,7 @@ where = [""]
 include = ["torchtune*", "recipes*"]
 
 [tool.setuptools.package-data]
-recipes = ["configs/*.yaml", "configs/*/*.yaml"]
+recipes = ["configs/*.yaml", "configs/*/*.yaml", "configs/*/*/*.yaml"]
```

Comment:
Can we just use something like `configs/**.yaml` here?

Reply:
it should be `configs/**/*.yaml` to match any number of subdirectories

Reply:
I think we already landed the existing version in another PR. Also after thinking about it more maybe it's better to be explicit here anyways